addons/binding/org.openhab.binding.lutron/ESH-INF/thing/keypads.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
There are many different button configurations for the keypads.  My thought was to have a generic keypad thing type with all the buttons that can be used for any keypad, and it would be up to the user to know which channels could be bound.  This is what's in the pull request.

My plan was to also include separate thing types for each button configuration that's sold, e.g. if someone has a 5BRL, they could configure the 5BRL thing type and it would only have the appropriate channels defined.  I didn't get a chance to do this, but the idea was to have all these keypad thing types in a separate file.

I can merge it into thing-types.xml if you prefer.